mws/resource_mws_network_connectivity_config.go

```diff
@@ -0,0 +1,92 @@
+package mws
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+// NewNetworkConnectivityConfigAPI creates NetworkConnectivityConfigAPI instance from provider meta
+func NewNetworkConnectivityConfigAPI(ctx context.Context, m any) NetworkConnectivityConfigAPI {
```

Comment:
@grayluck we merged the latest go sdk, which should have the NetworkConnectivity API